adcm_aio_client/objects/_common.py

```diff
@@ -22,65 +22,213 @@
 
 
 class Deletable(WithProtectedRequester, AwareOfOwnPath):
+    """
+    Mixin for objects that can be deleted
+    """
+
     async def delete(self: Self) -> None:
+        """
+        Delete object from ADCM. Object can't be restored and it's node accessor must have 'Deletable' mixin
```

Comment:
`and it's node accessor must have 'Deletable' mixin` -- strange tech detail, imo not relevant to client user

in other cases as well

maybe better to give a short detail about what is it (like in here with "Object can't be restored")